uint16 gpioId
---
bool hasSucceeded